src/material-experimental/mdc-core/option/optgroup.html

```diff
@@ -0,0 +1,8 @@
+<label
+  class="mat-mdc-optgroup-label"
+  [class.mdc-list-item--disabled]="disabled"
+  [id]="_labelId">
+  <span class="mdc-list-item__text">{{ label }} <ng-content></ng-content></span>
+</label>
+
+<ng-content select="mat-option, ng-container"></ng-content>
```

Comment:
Hmm, do we include `ng-container` in the existing version too? Feels kinda yucky, but I can see why we'd do that

Reply:
That's what we have in the current version. It's there, because sometimes people wrap multiple options in an `ng-container` so they can use a single `ngIf` rather than having to repeat it on each option.

Reply:
Yeah I figured, wish Angular could fix it somehow instead of us trying to work around it. It works in this case because we can pretty much assume we know where the user is trying to put the `ng-container`, but its not always so clear